src/esp/sensor/VisualSensor.h

```diff
@@ -23,10 +23,17 @@ namespace sensor {
 using Mn::Math::Literals::operator""_degf;
 
 struct VisualSensorSpec : public SensorSpec {
-  float ortho_scale = 0.1f;
-  vec2i resolution = {128, 128};        // height x width
-  std::string encoding = "rgba_uint8";  // For rendering colors in images
-  bool gpu2gpuTransfer = false;         // True for pytorch tensor support
+  vec2i resolution = {128, 128};  // height x width
+  int channels = 4;
```

Comment:
I realize you're just moving this field from elsewhere, but can you add a comment explaining its usage?